Makefile

```diff
@@ -8,7 +8,6 @@ else
 export GOPATH := $(shell go env GOPATH)
 GOPATH1 := $(firstword $(subst :, ,$(GOPATH)))
 endif
-export GOPROXY := direct
```

Comment:
this affects everywhere we use makefile (gha, jenkins, circleci, etc.). How has this been tested?